simpleGravitySim/simpleGravity.jl

```diff
@@ -0,0 +1,31 @@
+module simpleGravity
+
+using Trixi, OrdinaryDiffEq, LinearAlgebra, ForwardDiff;
+
+struct gravEq1D <: Trixi.AbstractEquations{1 #= number of spatial dimensions =#,
+                                        2 #= number of primary variables, (Φ, q1)=#}
+end
+
+@inline Trixi.flux(u, orientation, equation::gravEq1D) = SVector(0, -u[2])
```

Comment:
`u[1]` is you scalar variable, which you want to change. Try `SVector(-u[1], 0)` as a simple advection equation.